src/com/projectkorra/projectkorra/firebending/combustion/Combustion.java

```diff
@@ -1,5 +1,6 @@
 package com.projectkorra.projectkorra.firebending.combustion;
 
+import com.projectkorra.projectkorra.region.RegionProtection;

```

Comment:
projectkorra import has to be in projectkorra imports' section I guess? (Did it today in my PR :3)